### PR TITLE
Create Hello_world.c

### PR DESCRIPTION
**Pull request by AmanRana10**
Created a basic program in C.

### CODE REVIEW

File: 2021/C/BestPractice/Hello_world.c [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#include<stdio.h>
```
Comment (Vitaljaz) [inline 2021/C/BestPractice/Hello_world.c:1]: need space

File: 2021/C/BestPractice/Hello_world.c [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+int main()
+{
+  printf("Hello World");
```
Comment (Vitaljaz) [inline 2021/C/BestPractice/Hello_world.c:5]: need \n

File: 2021/C/BestPractice/Hello_world.c [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+}
```
Comment (Vitaljaz) [inline 2021/C/BestPractice/Hello_world.c:6]: add `return 0;`

**Review by Vitaljaz — CHANGES_REQUESTED** [review]
Need refactoring.

File: 2021/C/BestPractice/Hello_world.c [hunk 4]
```diff
@@ -1,6 +1,7 @@
-#include<stdio.h>
+ #include<stdio.h>
```
Comment (Vitaljaz) [inline 2021/C/BestPractice/Hello_world.c:1]: Remove extra spaces at the beginning of lines

**Review by Vitaljaz — CHANGES_REQUESTED** [review]
Remove extra spaces at the beginning of lines

**Comment by Vitaljaz** [comment]
@AmanRana10 nice! You need star this repo.

**Comment by AmanRana10** [comment]
done @Vitaljaz

**Comment by Vitaljaz** [comment]
@AmanRana10 good job.

**Pull request merged**